python/example/network_ring.py

```diff
@@ -32,31 +32,25 @@ def make_cable_cell(gid):
     b2 = tree.append(b0, arbor.mpoint(50, 0, 0, 1), arbor.mpoint(50+50/sqrt(2), -50/sqrt(2), 0, 1), tag=3)
 
     # Associate labels to tags
-    labels = arbor.label_dict()
-    labels['soma'] = '(tag 1)'
-    labels['dend'] = '(tag 3)'
-
-    # (2) Mark location for synapse at the midpoint of branch 1 (the first dendrite).
-    labels['synapse_site'] = '(location 1 0.5)'
-    # Mark the root of the tree.
-    labels['root'] = '(root)'
+    labels = arbor.label_dict({
+        'soma': '(tag 1)',
+        'dend': '(tag 3)',
+        # (2) Mark location for synapse at the midpoint of branch 1 (the first dendrite).
+        'synapse_site': '(location 1 0.5)',
+        # Mark the root of the tree.
+        'root': '(root)',})
```

Comment:
```suggestion
    labels = arbor.label_dict({
        'soma': '(tag 1)',
        'dend': '(tag 3)',
        # (2) Mark location for synapse at the midpoint of branch 1 (the first dendrite).
        'synapse_site': '(location 1 0.5)',
        # Mark the root of the tree.
        'root': '(root)',
    })
```

Again, I find closing symbols should be aligned with their opening symbols. With the trailing comma, it is also PEP8 compliant.

Reply:
Black is coming, see  #1906